packages/enoki-connect/package.json

```diff
@@ -45,9 +45,12 @@
 		"vitest": "^3.2.4"
 	},
 	"dependencies": {
+		"@fontsource/poppins": "^5.2.6",
```

Comment:
I don't love having this use the Enoki Connect font, maybe we can just use the system font for the text in dapps?

Reply:
I will remove it. Maybe I can follow up with a way to customise the styling